### PR TITLE
Reflow code for better/more consistent layout

### PR DESCRIPTION
**Pull request by minchinweb**
This pull request should not change any of the actual code, but adjusts the vertical alignment of most of the files, in the hope that they're more pleasing to look at.

Thanks for maintaining this!

### CODE REVIEW

File: GameData/002_CommunityPartsTitles/Localization/CPT_OpenCockpit_loc.cfg [hunk 1]
```diff
@@ -29,9 +29,9 @@ Localization
 
 		#LOC_nesdOC_title_OCovhSmall =  Wing Overhead panel Type A (0.5x0.5) (OC)  // Overhead panel Type A (OC)
 		#LOC_nesdOC_title_OCovhBig   =  Wing Overhead panel Type B (1x0.8) (OC)    // Overhead panel Type B (OC)
-		#LOC_nesdOC_title_OCovhBig2  =  Wing Overhead panel Type C (1x0.9) (OC)   // Overhead panel Type C (OC)
```
Comment (yalov) [inline GameData/002_CommunityPartsTitles/Localization/CPT_OpenCockpit_loc.cfg:32]: Why comment is removed?

File: GameData/002_CommunityPartsTitles/Localization/CPT_OpenCockpit_loc.cfg [hunk 2]
```diff
@@ -29,9 +29,9 @@ Localization
...
+		#LOC_nesdOC_title_OCovhBig2  =  Wing Overhead panel Type C (1x0.9) (OC)
```
Comment (yalov) [inline GameData/002_CommunityPartsTitles/Localization/CPT_OpenCockpit_loc.cfg]: Why comment is removed?

Comment (minchinweb) [reply GameData/002_CommunityPartsTitles/Localization/CPT_OpenCockpit_loc.cfg]: That's an oops. Good catch. I've re-added the comment.

**Comment by yalov** [comment]
thanks

**Pull request merged**